zerver/lib/actions.py

```diff
@@ -5127,6 +5129,23 @@ def do_update_user_custom_profile_data(user_profile: UserProfile,
                 "rendered_value": field_value.rendered_value,
                 "type": field_value.field.field_type})
 
+def do_check_remove_custom_profile_field_value(user_profile: UserProfile,
```

Comment:
Rename this to `check_remov_custom_...` (i.e. drop `do_` from the name) -- we generally reserve the name `do_foo` for functions that aren't expected to fail due to user input errors, which this function isn't, and use `check_foo` as the name for functions like this one.

Reply:
Oh, I didn't know that. Got it!